api/data_refinery_api/views.py

```diff
@@ -1001,8 +1013,12 @@ def _get_job_stats(cls, jobs, range_param):
                                       created_at__gt=JOB_CREATED_AT_CUTOFF)),
         )
         # via https://stackoverflow.com/questions/32520655/get-average-of-difference-of-datetime-fields-in-django
-        result['average_time'] = jobs.filter(start_time__isnull=False, end_time__isnull=False, success=True).aggregate(
-                average_time=Avg(F('end_time') - F('start_time')))['average_time']
+        result['average_time'] = jobs.filter(start_filter).filter(
+            start_time__isnull=False,
+            end_time__isnull=False,
+            success=True).aggregate(
```

Comment:
This is my preferred style for method chaining across lines in python, not sure that's gospel or anything but I think we're moderately consistent about it: https://github.com/AlexsLemonade/refinebio/blob/dev/workers/data_refinery_workers/downloaders/geo.py#L320

Reply:
yup! 👍